src/Rings/mpolyquo-localizations.jl

```diff
@@ -2267,10 +2267,10 @@ If the localization is at a point, a minimal set of generators is returned.
   L = localized_ring(Q)
 
   J = pre_image_ideal(I)
-  return filter(!iszero, Q.(small_generating_set(J)))
+  return unique!(filter(!iszero, Q.(small_generating_set(J))))
```

Comment:
Just out of curiosity: Do you have an small example, where the `unique!` is indeed necessary?

Reply:
For quotient rings the small generating set is computed over the polynomial ring. Then generators might fall together under the reduction map.